lib/learn_test/strategies/jasmine.rb

```diff
@@ -0,0 +1,153 @@
+require 'crack'
+require 'erb'
+require 'yaml'
+require 'json'
+
+require_relative 'jasmine/phantom_checker'
+require_relative 'jasmine/initializer'
+
+module LearnTest
+  module Strategies
+    class Jasmine < LearnTest::Strategy
+      def service_endpoint
+        '/e/flatiron_jasmine/build/ironboard'
+      end
+
+      def detect
+        runner.files.include?('requires.yml')
+      end
+
+      def check_dependencies
+        LearnTest::Jasmine::PhantomChecker.check_installation
```

Comment:
this needs to only run if `options[:skip]` isn't set (otherwise it'll break on nitrous)